Client/Frontend/Browser/Certificate Viewer/CertificateViewController.swift

```diff
@@ -0,0 +1,527 @@
+        do {
+            let absolute_oid: [UInt8] = try absolute_oid_to_oid(oid: oid)
+            return Data(bytes: absolute_oid, count: absolute_oid.count)
+        } catch {
+            return Data()
+        }
+    }
+
+    static func oid_to_absolute_oid(oid: Data) -> String {
+        do {
+            return try oid_to_absolute_oid(oid: oid.getBytes())
+        } catch {
+            return String()
+        }
+    }
+}
+
+struct CertificateTitleView: View {
+    let isRootCertificate: Bool
+    let commonName: String
+    
+    var body: some View {
+        HStack(alignment: .center, spacing: 15.0) {
+            Image(uiImage: isRootCertificate ? #imageLiteral(resourceName: "Root") : #imageLiteral(resourceName: "Other"))
+            VStack(alignment: .leading, spacing: 10.0) {
+                Text(commonName)
+                    .font(.system(size: 16.0, weight: .bold))
+            }
+        }.background(Color(UIColor.secondaryBraveGroupedBackground))
+    }
+}
+
+struct CertificateKeyValueView: View, Hashable {
+    let title: String
+    let value: String?
+    
+    init(title: String, value: String? = nil) {
+        self.title = title
+        self.value = value
+    }
+    
+    var body: some View {
+        HStack(alignment: .center, spacing: 12.0) {
+            Text(title)
+                .font(.system(size: 12.0))
+            Spacer()
+            if let value = value, !value.isEmpty {
+                Text(value)
+                    .lineLimit(nil)
+                    .fixedSize(horizontal: false, vertical: true)
+                    .font(.system(size: 12.0, weight: .medium))
+            }
+        }
+    }
+}
+
+struct CertificateSectionView<ContentView>: View where ContentView: View {
+    let title: String
+    let values: [ContentView]
+    
+    var body: some View {
+        Section(header: Text(title)
+                    .font(.system(size: 12.0))) {
+            
+            ForEach(values.indices, id: \.self) {
+                values[$0].listRowBackground(Color(UIColor.secondaryBraveGroupedBackground))
+            }
+        }
+    }
+}
+
+struct CertificateView: View {
+    @EnvironmentObject var model: BraveCertificate
+    
+    var body: some View {
+        VStack {
+            CertificateTitleView(isRootCertificate:
+                                    model.value.isRootCertificate,
+                                 commonName: model.value.subjectName.commonName).padding()
+            
+            if #available(iOS 14, *) {
+                List {
+                    content
+                }
+                .listStyle(InsetGroupedListStyle())
+                .frame(maxHeight: .infinity)
+                .environmentObject(model)
+            } else {
+                List {
+                    content
+                }
+                .listStyle(GroupedListStyle())
+                .frame(maxHeight: .infinity)
+                .environmentObject(model)
+            }
```

Comment:
Can remove the 14 check, we are 14+, `.frame(maxHeight: .infinity)` is likely redundant for List.  `environmentObject` is also not needed as `content` belongs to this view